client/driver/executor/test_harness_test.go

```diff
@@ -79,7 +132,7 @@ func Executor_Start_Invalid(t *testing.T, command buildExecCommand) {
 }
 
 func Executor_Start_Wait_Failure_Code(t *testing.T, command buildExecCommand) {
-	e := command("/bin/date", "-invalid")
+	e := command(os.Args[0], "fail")
```

Comment:
What do you think about putting `os.Args[0]` into a a variable. It could then be documented and be given a more useful name to make it clear that we are calling ourselves.